debugger/src/main/resources/META-INF/debugger-only.xml

```diff
@@ -1,15 +1,33 @@
-<idea-plugin xmlns:xi="http://www.w3.org/2001/XInclude">
-    <depends>com.intellij.cidr.debugger</depends>
```

Comment:
As I see, now the plugin is not dependent on `com.intellij.cidr.debugger`. But we still have a corresponding check in `org.rust.debugger.DebuggingIntegrationKt#isDebuggingIntegrationEnabled`. Does it work fine even with 2021.3?

Reply:
It works fine. As a previously, the plugin depends on `com.intellij.modules.cidr.debugger`. This module is provided by `com.intellij.cidr.debugger` plugin in CLion/AppCode/Rider 2022.1 and by native debugging support plugin in IDEA/PyCharm/GoLand/RubyMine. And `isDebuggingIntegrationEnabled` check the same thing in practice.

Actually, we can check `com.intellij.modules.cidr.debugger` "plugin" since the platform uses such platform module ids as synonyms to the actual plugins providing the module. But I've found out this fact right now 😄 

And since #8709 drops `isDebuggingIntegrationEnabled` at all so I won't fix `isDebuggingIntegrationEnabled`

Reply:
Interesting, OK then 👌